modules/calib3d/test/test_solvepnp_ransac.cpp

```diff
@@ -837,6 +837,92 @@ TEST(Calib3d_SolvePnPRansac, double_support)
     EXPECT_LE(cvtest::norm(t, Mat_<double>(tF), NORM_INF), 1e-3);
 }
 
+TEST(Calib3d_SolvePnPRansac, bad_input_points)
+{
+    vector<Point3f> pts3d;
+    vector<Point2f> pts2d;
+
+    pts3d = {
```

Comment:
See the buildbot (https://pullrequest.opencv.org/#/summary/opencv) errors.

This requires C++11 features.

Reply:
Fixed now.